tools/dev-doctor/rootcmd.go

```diff
@@ -50,6 +50,12 @@ func rootCmdRun(cmd *cobra.Command, args []string) {
 	checks := []check{
 		osArchCheck{},
 		unameCheck{},
+		&binaryCheck{
+			name:          "make",
+			ifNotFound:    checkError,
+			versionArgs:   []string{"--version"},
+			versionRegexp: regexp.MustCompile(`GNU\s+Make\s+(\d+\.\d+\.\d+)`),
```

Comment:
This check fails for me with GNU `make` 4.3:

```
% make --version
GNU Make 4.3
Built for x86_64-pc-linux-gnu
Copyright (C) 1988-2020 Free Software Foundation, Inc.
License GPLv3+: GNU GPL version 3 or later <http://gnu.org/licenses/gpl.html>
This is free software: you are free to change and redistribute it.
There is NO WARRANTY, to the extent permitted by law.
```

Changing this to
```suggestion
			versionRegexp: regexp.MustCompile(`GNU\s+Make\s+(\d+\.\d+\S*)`),
```
fixes it for me.

Reply:
Thanks, PR updated with your change.